README.md

```diff
@@ -19,7 +20,7 @@ provided for getting started quickly.
 License
 -------
 
-Copyright 2013-2018 pyMOR developers and contributors. All rights reserved.
+Copyright 2013--2019 pyMOR developers and contributors. All rights reserved.
```

Comment:
Why the double dash? At least Github seems to render this as two individual dashes, which looks odd to me.